tests/testapp/models.py

```diff
@@ -13,6 +13,7 @@ def frozen_uuid_generator():
 
 
 class NumberModel(hc_models.HerokuConnectModel):
+    sf_access = 'read_write'
```

Comment:
Are you sure you need to change both of them?

Reply:
Both the models are used write(save/update), hence it should have read_write sf_access .

Reply:
ah I see